script.js

```diff
@@ -43,14 +46,12 @@ function setup() {
       return obj.summary.includes(e.target.value) || obj.name.includes(e.target.value)
```

Comment:
Is it case insensitive? If the user accidentally pressed Caps Lock, will the search bar still work?